test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -11,7 +11,10 @@ TEST_NO_NSPAWN=1
 test_append_files() {
     (
         local workspace="${1:?}"
-        dracut_install busybox
+
+        # On openSUSE the static linked version of busybox is named "busybox-static".
+        busybox=$(type -P busybox-static || type -P busybox)
+        inst_simple $busybox $(dirname $busybox)/busybox
```

Comment:
why inst_simple rather than dracut_install like before?

Reply:
Also, please keep it shellcheck-happy, i.e.

```bash
        busybox="$(type -P busybox-static || type -P busybox)"
        inst_simple "$busybox" "$(dirname "$busybox")/busybox"
```

Reply:
@bluca because the target filename needs to be specified in case the source filename is "busybox-static" and I don't think `dracut_install` can do that. Also busybox is statically linked so a "dumb" coy should be safe.